OrbitQt/AccessibilityAdapter.cpp

```diff
@@ -0,0 +1,269 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/*
```

Comment:
Should some of this documentation be slightly updated to reflect the new module structure?

Reply:
They absolutely should, and I completely forgot about this, thanks. Updated the diagrams and the documentation to reflect the change in module structure and naming.